tensorbay/cli/auth.py

```diff
@@ -57,8 +58,12 @@ def _implement_auth(  # pylint: disable=too-many-arguments
     else:
         error(f'Invalid argument "{arg1}"')
 
+    if is_accesskey(arg1):
+        _check_auth(arg1, arg2, obj["profile_name"])
+    else:
+        _check_auth(arg2, arg1, obj["profile_name"])
```

Comment:
Is it possible to move the check into `_update_profile`?
And is it necessary to use `is_accesskey`?